### PR TITLE
Qualification tool: Remove "unsupported" word for nested complex types

### PR DESCRIPTION
**Pull request by nartal1**
This fixes https://github.com/NVIDIA/spark-rapids/issues/3764 .

This is a minor PR to remove "Unsupported" word for nested complex types.  Updated the output files to match the new format.

### CODE REVIEW

**Comment by nartal1** [comment]
build

**Review by tgravescs — DISMISSED** [review]
please watch build to see if we need to update qualification IT tests

File: docs/spark-qualification-tool.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ Here is a brief description of each of column that is in the CSV:
 15. Unsupported Write Data Format: Reports the data format which we currently don’t support, i.e.
     if the result is written in JSON or CSV format.
 16. Complex Types: Looks at the Read Schema and reports if there are any complex types(array, struct or maps) in the schema.
-17. Unsupported Nested Complex Types: Nested complex types are complex types which
+17. Nested Complex Types: Nested complex types are complex types which
```
Comment (tgravescs) [inline docs/spark-qualification-tool.md:383]: actually if we aren't saying these are unsupported do we need more text here stating why we are reporting these separately?  Should we say something like not all nested types supported, please see the docs for the release you are using?
@viadea ideas here to make sure its clear why this field is here?

Comment (viadea) [reply docs/spark-qualification-tool.md:383]: I think we do claim that there is 
```
Here the Problematic Duration is zero but please keep in mind that we are only able to detect certain issues. This currently includes some UDFs, some decimal operations and nested complex types. 
```
So I think this should be fine.

Comment (nartal1) [reply docs/spark-qualification-tool.md:383]: @tgravescs Please let me know if you think more context is required here in the docs. Or the one which @viadea has posted would be sufficient.

Comment (tgravescs) [reply docs/spark-qualification-tool.md:383]: ok its fine.

**Review by tgravescs — APPROVED** [review]

**Pull request merged**